lti_consumer/lti_consumer.py

```diff
@@ -740,6 +806,85 @@ def student_view(self, context):
         fragment.initialize_js('LtiConsumerXBlock')
         return fragment
 
+    has_author_view = True
+
+    def author_view(self, context):
```

Comment:
Implemented an `author_view` since a lot of data needs to be set up in the tool for LTI 1.3 to work.